### PR TITLE
Don't panic on scopes outside of functions, or calls with no args

### PR DESCRIPTION
**Pull request by reillywatson**

### CODE REVIEW

**Comment by daisuzu** [comment]
@reillywatson 
Thank you for PR!
Which code has panic? And can it be added to the test?

**Comment by reillywatson** [comment]
Added some test cases! The added b/context would panic because of a call with no args on line 18 of that file before, and the change to a.go would panic on the scope check.

File: ctxscope/testdata/src/a/a.go [hunk 1]
```diff
@@ -25,6 +25,8 @@ func (datastoreInterface) Delete(c context.Context, key interface{}) error {
 
 var datastore = datastoreInterface{}
 
+var ctx, _ = context.WithCancel(context.TODO())
```
Comment (daisuzu) [inline ctxscope/testdata/src/a/a.go]: I think there is no need to define as `ctx`, instead with `_`

Comment (reillywatson) [reply ctxscope/testdata/src/a/a.go]: Good point, updated!

**Comment by daisuzu** [comment]
Great!
Many thanks!!

**Pull request merged**